### DISCUSSION TITLE
Very Bad

### DISCUSSION THREAD

**Discussion by mmedhat2** [woocommerce/woocommerce#51459] [Checkout Flow]
pls this is so frustrating the previous version was better

**Comment by opr** [comment]
@mmedhat2 thanks for your feedback. Would you be willing to share what makes the shortcode version of Cart/Checkout better for you? Your feedback will help us improve the experience.

If you want to revert, please see the documentation: [Reverting to the classic Cart and Checkout](https://woocommerce.com/document/woocommerce-store-editing/customizing-cart-and-checkout/#reverting-to-the-classic-cart-and-checkout)
